modules/ztp-configuring-disk-partitioning.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_module-type: PROCEDURE
+[id="ztp-configuring-disk-partitioning_{context}"]
+= Configuring disk partitioning with SiteConfig
+
+You can use the `SiteConfig` YAML file to generate the `MachineConfig` CR used for disk partitioning. You need to to modify values in the `SiteConfig` CR to reflect dependencies on the underlying disk.
+The values must be established prior to installation.
```

Comment:
This either needs an empty line above it, or it can be added to the statement above on line 9. 

Optionally, it seems high priority enough to warrant an `[IMPORTANT]` note, right? 

And what values are we referring to? `SiteConfig` or ``MachineConfig`?

Reply:
Went with "Prior to installation, you need to to modify values in the `SiteConfig` CR to reflect dependencies on the underlying disk."
MachineConfig is generated using SiteConfig.